File: src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/importering/autoimport/statistikk/SykefraværsstatistikkIntegrasjon.java
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.importering.autoimport.statistikk;

import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;

public abstract class SykefraværsstatistikkIntegrasjon {

  public static final String ARSTALL = "arstall";
  public static final String KVARTAL = "kvartal";

  final NamedParameterJdbcTemplate namedParameterJdbcTemplate;

  public SykefraværsstatistikkIntegrasjon(NamedParameterJdbcTemplate namedParameterJdbcTemplate) {
    this.namedParameterJdbcTemplate = namedParameterJdbcTemplate;
  }
}

File: src/main/kotlin/no/nav/arbeidsgiver/sykefravarsstatistikk/api/infrastruktur/database/SykefraværsstatistikkIntegrasjon.kt
package no.nav.arbeidsgiver.sykefravarsstatistikk.api.infrastruktur.database

import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate

abstract class SykefraværsstatistikkIntegrasjon(@JvmField val namedParameterJdbcTemplate: NamedParameterJdbcTemplate) {
    companion object {
        const val ARSTALL = "arstall"
        const val KVARTAL = "kvartal"
    }
}
